charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 {{ include "azuredisk.labels" . | indent 2 }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.controller.replicas }}
```

Comment:
what's the default value now? this csi driver would run e2e test on this chart config, `3` is ok for me.